salt/utils/vt.py

```diff
@@ -61,31 +63,36 @@ class TerminalException(Exception):
     """
 
 
-# ----- Cleanup Running Instances ------------------------------------------->
-# This lists holds Terminal instances for which the underlying process had
-# not exited at the time its __del__ method got called: those processes are
-# wait()ed for synchronously from _cleanup() when a new Terminal object is
-# created, to avoid zombie processes.
-_ACTIVE = []
-
-
-def _cleanup():
```

Comment:
This function was removed, but https://github.com/saltstack/salt/blob/ff31d08079d2100f49c6471461bc22724bf6c392/salt/utils/vt.py#L129 relies on it

Reply:
fixed